scripts/test-runner.js

```diff
@@ -171,31 +169,37 @@ const stopRC = async () => {
 if (process.argv.length === 3 || process.argv.length === 4) {
     if (process.argv[2] === 'unit') {
         if (process.argv.length === 4) {
-            testCommand = `node node_modules/mocha/bin/mocha --recursive -g "${process.argv[3]}" "test/unit/**/*.js"`;
+            testCommand = `node node_modules/mocha/bin/mocha -g "${process.argv[3]}" "test/unit/**/*.js" && `;
```

Comment:
Is the `&&` at the end required? Also, why don't we need `--recursive` anymore?

Reply:
no it is not, thanks. I think recursive is only needed when we don't give glob pattern. I don't use it in my day-to-day mocha invocations and it is fine. So deleted since it is not needed.